docs/developers/metrics-configuration.md

```diff
@@ -0,0 +1,74 @@
+## Metrics configuration and how to import metrics of [Prebid server](https://docs.prebid.org/prebid-server/versions/pbs-versions-go.html) to [Grafana](https://grafana.com/) through [Prometheus](https://prometheus.io/) stack.
+     
+#### These are the parameters that are capable for metrics collection 
+
+- PBS_METRICS_PROMETHEUS_PORT=9100 - default is 0.
+
+If PBS_METRICS_PROMETHEUS_PORT is null, your [Prebid server](https://docs.prebid.org/prebid-server/versions/pbs-versions-go.html) won't collect any metrics.
+  
+- PBS_METRICS_PROMETHEUS_NAMESPACE=prebid - default is empty.
+  
+PBS_METRICS_PROMETHEUS_NAMESPACE - for your unique metrics
+  
+- PBS_METRICS_PROMETHEUS_SUBSYSTEM=server - default is empty.
+  
+PBS_METRICS_PROMETHEUS_NAMESPACE - for your deep unique metrics
+  
+- PBS_METRICS_DISABLED_METRICS_ADAPTER_CONNECTIONS_METRICS=false - default is true.
+
+If PBS_METRICS_DISABLED_METRICS_ADAPTER_CONNECTIONS_METRICS is true your [Prebid server](https://docs.prebid.org/prebid-server/versions/pbs-versions-go.html) won't collect any adapter metrics
+
+#### If you're going to get metrics though [Prometheus](https://prometheus.io/) and [Prometheus](https://prometheus.io/) stack has been already installed, you have several options, please chose one:
+
+- change environments into code (bad way).
+ 
+- create pbs.yaml if it hasn't been already created (It works, but not flexible).
+
+- input the parameters into global environments (Highly recommend you to use this way also it's reliable).
+
+#### Finally, when you fill in your credentials of metrics, please, run:
+
+```bash
+./validate.sh
+```
+
+#### Make sure that an application returns any metrics - `http://localhost:9100/metrics`, if not, please check your names of environment and recompile again.
```

Comment:
Nitpick: also available at `http://localhost:9100` as Prometheus runs within separate service, not endpoint

Reply:
I've fixed it.